crates/circuit/src/operations.rs

```diff
@@ -286,6 +288,27 @@ impl Operation for OperationRef<'_> {
             Self::Unitary(unitary) => unitary.directive(),
         }
     }
+    #[inline]
```

Comment:
Could you add a docstring that clearly states that this will return `None` if the operation is not 1q? Calling something like `StandardGate::CXGate.get_mat_static_1q(&[])` still seems quite strange to me, so a comment would help 🙂

Reply:
Hmmm I can't find the docstring, did you push it or did I just not find it?

Reply:
Oops, indeed, I forgot to push this, done :).